docker-compose.yml

```diff
@@ -41,4 +41,4 @@ services:
     environment:
       - DB_UP_PASSWORD="${DB_UP_PASSWORD:-dbuserprofile}"
     ports:
-      - 5432:5432
\ No newline at end of file
+      - 5432
```

Comment:
Please revert this back to what was there. Otherwise there won't be any external port available. On your local machine you need to set your POSTGRES_PORT=5432 rather than hack the config file to work locally as you have done so here, also you should not use 5432 rather pick an unused port number that won't conflict with prd-professional-api.

Reply:
Done.

Reply:
I checked 5432 with both API docker works fine.